build.gradle

```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.2.71'
+    ext.kotlin_version = '1.3.10'
```

Comment:
@ebruersoy is this `kotlin_version` backward compatible?

Reply:
yes

Reply:
maybe we can migrate with another task.